SimPEG/Examples/MT_1D_ForwardAndInversion.py

```diff
@@ -1,9 +1,12 @@
-import SimPEG as simpeg
-import numpy as np
-import SimPEG.MT as MT
-from scipy.constants import mu_0
 import matplotlib.pyplot as plt
 
+
```

Comment:
Two things with this example 
- there is an overflow warning that is thrown, 
- and it doesn't look like the plots are working. 
@grosenkj : could you take a look? 
![image](https://cloud.githubusercontent.com/assets/6361812/18839735/e8577ffa-83c0-11e6-99f4-a216a85d6ffb.png)